src/com/createsend/models/journeys/JourneyEmailDetailWithGeoBase.java

```diff
@@ -0,0 +1,11 @@
+package com.createsend.models.journeys;
+
+public class JourneyEmailDetailWithGeoBase {

```

Comment:
The clicks, bounces and unsubscribes all derive from this base. However, the API doc doesn't mention that geo information is part of these details. Could you please confirm and if applicable, update the API doc?